inscriptions/ISic003203.xml

```diff
@@ -189,14 +194,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#FP">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <persName type="attested"><name type="nomen"><supplied reason="lost">Ma</supplied>nliae</name></persName> <persName type="attested"><name type="praenomen"><expan><abbr>T</abbr><ex>iti</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>iliae</ex></expan></w>
```

Comment:
nomen type on T(iti) is 'patronym' rather than 'praenomen', because it here appears as the patronym for Manlia; however, we still put the separate persName on it, as you have done here, so that we can capture the reference to the individual called Titus.